api/app/signals/apps/email_integrations/reporter/mail_actions.py

```diff
@@ -250,9 +246,22 @@ def _get_mail_context(self, signal: Signal, mail_kwargs: dict):
     def _mail(self, signal: Signal, mail_kwargs: dict):
         context = self._get_mail_context(signal=signal, mail_kwargs=mail_kwargs)
 
-        message = loader.get_template(mail_kwargs['templates']['txt']).render(context)
-        html_message = loader.get_template(mail_kwargs['templates']['html']).render(context)
-        subject = mail_kwargs['subject'].format(signal_id=signal.id)
+        try:
```

Comment:
Nice, maybe add a note that in time we want to move all templates to the database. Then in time we can remove the rendering of templates from files.

Reply:
Done 👍